test/simple/inventory_14_2.1/hosts.libvirt.yml

```diff
@@ -13,33 +13,40 @@ all:
     postgres_cluster:
       hosts:
         monitor:
-          ansible_ssh_host: "127.0.0.1"
-          ansible_ssh_port: "2200"
+#          ansible_ssh_host: "127.0.0.1"
+#          ansible_ssh_host: "10.0.0.10"
+#          ansible_ssh_port: "2200"
+#          ansible_ssh_port: "22"
           ansible_ssh_user: "vagrant"
-          ansible_ssh_private_key_file: "{{ playbook_dir }}/test/simple/.vagrant/machines/monitor/virtualbox/private_key"
+          ansible_ssh_private_key_file: "/home/fred/code/wannabegeekster/pg_auto_failover_ansible/test/simple/.vagrant/machines/monitor/libvirt/private_key"
```

Comment:
can we make this a relative path?

Reply:
same for the other occurences in this file

Reply:
Apologize for that.  Should have grepped for that for sure.  It was very late after a very long day. I definitely did not want that in a public repo.

Reply:
no worries, that's what a code review is for.